guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -12,13 +12,14 @@ To create a file type repository in a directory on a remote server, see xref:Cre
 
 To create a file type repository in a local directory, complete the following procedure:
 
-. Ensure the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. Ensure the Utils repository is enabled.
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# subscription-manager repos --enable={RepoRHEL7Server} \
```

Comment:
Note this is marked to cherry pick to 3.1 where EL7 is supported for all platforms. That can cause user confusion.

Reply:
Does that mean I (luckily) missed cherry-picking this to 3.1 and it actually needs to be adjusted first? cc @dubewarsagar

Reply:
this PR was created to be merged in 3.1 and 3.3 only. The original #1650 exists for other versions which needs to be adjusted before merging to master for 3.4 and above. The #1650 should have no instructions about EL 7 and thats what we discussed here in this thread: https://github.com/theforeman/foreman-documentation/pull/1650#discussion_r983278903

@maximiliankolb does this clarify your query?

Reply:
@dubewarsagar to be clear, I suggested there to open a separate PR for 3.1 since it's very different from 3.3. To be explicit in what I intended:
* Open a PR for master and picked into 3.4 that is essentially what this PR is now: no EL7 server instructions at all
* Open a PR for 3.3: EL7 server instructions for upstream
* Open a PR for 3.1: EL7 server instructions for all

Reply:
I was following slightly above [comment](https://github.com/theforeman/foreman-documentation/pull/1650#discussion_r983136681) so created this branch for 3.1 and 3.3. Also this branches base is updated recently from master to 3.3. So this PR we cannot use for 3.4. 

Now that this branch is merged into 3.3, can we:
* Use #1650 for 3.4 (merge only after I remove EL7 instructions)
* Open a new PR for 3.1: EL7 server instructions for all

Reply:
@ekohl 

I have created following new PRs to fix the respective versions. Kindly help to merge them and close this issue:

- For Foreman 3.4/Katello 4.6, created #1722 
- For Foreman 3.1/Katello 4.3, created #1723 

Kindly inform if anything is remaining on this one. 

Thank you :)